cmd/data.go

```diff
@@ -7,6 +7,8 @@ import "encoding/xml"
 type JsonHost struct {
 	Port  int `json:"port"`
 	Count int `json:"count"`
+	Username string `json:"username"`
```

Comment:
Please `gofmt` all changed files. This should indent source correctly.

Reply:
Was not aware of that, thanks.